src/lib/Builder/NotificationBuilder.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Builder;
+
+use Twig\Environment;
+
+/**
+ * Builder for notification messages, returns output from twig.
+ */
+class NotificationBuilder
```

Comment:
hum @bdunogier while working on https://jira.ez.no/browse/EZP-27476 will also need to handle Notification and we were also talking about that lately in https://github.com/ezsystems/hybrid-platform-ui/pull/30#discussion_r124520732 where we were thinking about a `Notification` value object.
So that's a hot topic where we (you :p) to synchronize